vevestaX/vevesta.py

```diff
@@ -301,6 +298,9 @@ def dump(self, techniqueUsed, filename=None, message=None, version=None, showMes
         df_messages = pandas.DataFrame(index=[1], data=data)
         df_messages = pandas.concat([messageData, df_messages], ignore_index=True)
 
+        self.__sampleSize=100 if self.__sampleSize>=100 else self.__sampleSize
+        self.__data=self.__data.sample(self.__sampleSize)
```

Comment:
change to  sampledData =self.__data.sample(self.__sampleSize). Write sampleData into the excel.